src/averageAges.js

```diff
@@ -15,11 +15,16 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const men = people
+    .filter((person) => (
+      century === undefined
+      || Math.ceil(person.died / 100) === century
+    ))
+    .filter((person) => person.sex === 'm');
```

Comment:
* Consider creating variables with meaningful names instead of using magic strings
* Follow the DRY principle: don't repeat yourself. You are using the sex check a lot of times. Consider creating a function for this purpose